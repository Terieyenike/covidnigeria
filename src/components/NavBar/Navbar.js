import { Div, Section } from './Navbar.style';
import Container from '../Container';

const Navbar = ({ handleChange }) => {
  return (
    <Section>
      <Container>
        <Div>
          <h1>Covid-19 in Nigeria</h1>
          <input
            aria-label='Search'
            placeholder={'Search by states'}
            onChange={handleChange}
          />
        </Div>
      </Container>
    </Section>
  );
};

export default Navbar;
