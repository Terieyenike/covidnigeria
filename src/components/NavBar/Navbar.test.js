// import { shallow } from 'enzyme';
import { create } from 'react-test-renderer';
import React from 'react';
import Navbar from './Navbar';

const navbar = create(<Navbar />);

it('expect to render the Navbar component', () => {
  expect(navbar.toJSON()).toMatchSnapshot();
});
