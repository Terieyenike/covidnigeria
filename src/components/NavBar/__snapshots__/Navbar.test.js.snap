// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render the Navbar component 1`] = `
<section
  className="sc-bdfBwQ NzFpA"
>
  <div
    className="sc-dlfnbm ivSpyO"
  >
    <div
      className="sc-gsTCUz bGCMpA"
    >
      <h1>
        Covid-19 in Nigeria
      </h1>
      <input
        aria-label="Search"
        placeholder="Search by states"
      />
    </div>
  </div>
</section>
`;
